src/lib/utils.js

```diff
@@ -1,5 +1,86 @@
+const path = require('path');
+const fs = require('fs');
+const moment = require('moment');
+const isEmpty = require('lodash/isEmpty');
+
 const UTILS = {
+  getUserDirectory() {
+    const homePath = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
+    return path.join(`${homePath}/geek-lab_local`);
+  },
+
+  readInternalCliFile(fileName) {
+    const homePath = UTILS.getUserDirectory();
+    const filePath = `${homePath}/${fileName}`;
+    try {
+      return JSON.parse(fs.readFileSync(filePath, 'utf8'));
+    } catch (e) {
+      console.log('Something went wrong, either it couldnt find the file, or its failing to parse it as json');
+      console.log(`Could not find folder or file ${filePath}`);
+      console.log(`Debug instructions: 
+      - check file for invalid characters
+      - install the tool again and it will generate everything you need if missing something`);
+      console.log(e);
+      throw new Error('Failed to read file');
+    }
+  },
+
+  writeInternalCliFile(fileName, data) {
+    const homePath = UTILS.getUserDirectory();
+    const filePath = `${homePath}/${fileName}`;
+    try {
+      fs.writeFileSync(filePath, JSON.stringify(data, null, '  '));
+    } catch (e) {
+      console.log(`something went wrong while attempting to write file ${filePath}`);
+      console.log(e);
+      throw new Error(`Failed to write file ${filePath}`);
+
+    }
+  },
+
+  readConfig(){
```

Comment:
im fine with `getConfig` just made it this way to keep it consistent with my other two methods (`read...` and `write...`)